LEGO1/lego/sources/roi/legoroi.cpp

```diff
@@ -58,8 +59,12 @@ ColorOverride g_colorOverride = NULL;
 TextureHandler g_textureHandler = NULL;
 
 // FUNCTION: LEGO1 0x100a81b0
-void LegoROI::FUN_100a81b0(const LegoChar* p_error, const LegoChar* p_name)
+// FUNCTION: BETA10 0x101898c0
+// FUNCTION: ALPHA 0x100bb1c0
+void LegoROI::FUN_100a81b0(const LegoChar* p_error, ...)
```

Comment:
The `...` didn't cause any regression as far as I can tell, but I'll double-check